.cirrus.yml

```diff
@@ -0,0 +1,579 @@
+    fi
+
+    # Clean up all packages
+    pkg delete --all --yes
+    pkg autoremove --yes
+    pkg clean --yes
+
+    if echo ${packagesite} | grep -q "https"
+    then
+      pkg install --yes ca_root_nss
+    fi
+
+    # Start using plugin repos
+    export REPOS_DIR=$repos_dir
+
+    echo "Fetching $name pkgs: $pkgs"
+    pkg fetch --dependencies --yes $pkgs
+
+    pkg delete --yes ca_root_nss || true
+
+    echo "Installing $name pkgs: $pkgs"
+    pkg install --no-repo-update --yes $pkgs
+
+    if [ -d "${plugin_dir}/overlay" ]
+    then
+      echo "Found overlay folder"
+      cp -r ${plugin_dir}/overlay/ /
+    fi
+
+    ${plugin_dir}/post_install.sh
+
+    if [ -f ${plugin_dir}/pre_update.sh ] && ! [ -x ${plugin_dir}/pre_update.sh ]
+    then
+      echo "pre_update.sh script not executable"
+      exit 1
+    fi
+
+    if [ -f ${plugin_dir}/post_update.sh ] && ! [ -x ${plugin_dir}/post_update.sh ]
+    then
+      echo "post_update.sh script not executable"
+      exit 1
+    fi
+
+backuppc_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "backuppc.json"
+
+bazarr_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "bazarr.json"
+
+bind_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-3
+  env:
+    PLUGIN_FILE: "bind.json"
+
+calibre-web_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "calibre-web.json"
+
+channels-dvr_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-3
+  env:
+    PLUGIN_FILE: "channels-dvr.json"
+
+clamav_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "clamav.json"
+
+deluge-pip_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "deluge-pip.json"
+
+dnsmasq_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-3
+  env:
+    PLUGIN_FILE: "dnsmasq.json"
+
+drupal8_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "drupal8.json"
+
+duplicati_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-3
+  env:
+    PLUGIN_FILE: "duplicati.json"
+
+emby_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "emby.json"
+
+emby-server-stable_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-2
+  env:
+    PLUGIN_FILE: "emby-server-stable.json"
+
+esphome_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
```

Comment:
```suggestion
        image_family: freebsd-12-2
```

Reply:
Sure thing, I was making this change as well on one of my test branches.